services/app-api/utils/auth/authorization.ts

```diff
@@ -92,3 +93,21 @@ export const hasPermissions = (
 
   return isAllowed;
 };
+
+export const hasAccess = (event: APIGatewayProxyEvent, reportType: string) => {
```

Comment:
The naming of this method is pretty interchangeable with hasPermissions() at a glance. Maybe hasReportAccess() like the var name you used, or updating this one to hasReport, and the other hasRole?